src/homepageExperience/components/steps/cli/InstallDependencies.tsx

```diff
@@ -120,6 +120,15 @@ sudo cp influxdb2-client-latest-linux-arm64/influx /usr/local/bin/
       {currentSelection === 'Mac' && (
         <>
           <h2 style={headingWithMargin}>Use Homebrew</h2>
+          <p style={{fontSize: '14px'}}>
```

Comment:
Not necessary to hold up this PR for this, but I'd like to see you follow up on this: we already style paragraph tags with a font size of 14px in the [this file](https://github.com/influxdata/ui/blob/master/src/homepageExperience/components/HomepageExperience.scss#L81) - I think we should make use of that scss file rather than having an inline style here.

Reply:
Hmm, tried that and for some reason it really messes with the sentence, breaks it into 3 parts that are all over the place